.github/workflows/link-check.yml

```diff
@@ -31,3 +33,19 @@ jobs:
           config-file: ".docs_mlc_config.json"
           folder-path: "docs"
           max-depth: 2
+  website-link-checker:
+    runs-on: ubuntu-latest
+    name: broken-link-check
+    steps:
+      - name: Check for broken links
+        id: link-report
+        uses: celinekurpershoek/link-checker@master
+        with:
+          # Required:
+          url: "https://www.featureform.com/"
+          # optional:
+          honorRobotExclusions: false
+          ignorePatterns: "*linkedin*"
```

Comment:
Can you explain this in a comment

Reply:
This is to exclude Linkedin links for checking since Linkedin returns an automatic 999 error to web scrapers.